concepts/pointers/introduction.md

```diff
@@ -0,0 +1,131 @@
+# Introduction
+
+Like many other languages, C++ has pointers.
+You already know `references` and pointers are similar, but think of them as a level closer to the inner workings of your computer.
```

Comment:
Here and in the rest of the PR: the use of backticks looks inconsistent. I'd suggest using backticks for code and italics or quotes for technical terms that you want to highlight.